end-to-end-computer-vision/steps/export_label_studio.py

```diff
@@ -58,7 +58,7 @@ def load_data_from_label_studio(
             "This step can only be used with the Label Studio annotator."
         )
 
-    if annotator and annotator._connection_available():
```

Comment:
did this lead to an error for you?

Reply:
yes, the function would fail to recognize that a connection is available. Removing the function showed that it was incorrectly failing the step

Reply:
ok then good to have this fixed